src/components/navbar.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function Navbar() {
+  const links = [
+    { id: 1, name: 'Rockets', path: '/' },
+    { id: 2, name: 'Missions', path: '/missions' },
+    { id: 3, name: 'My Profile', path: '/profile' },
+  ];
+
+  return (
+    <nav>
+      <div className="logo">
+        <div className="logo-img">
+          <img
+            src="https://cdn-icons-png.flaticon.com/512/3212/3212608.png"
+            alt="logo"
+            style={{ width: '100px', height: '100px' }}
+            className="logo-img"
+          />
+        </div>
+        <span className="logo-title">Space Travelers&apos; Hub</span>
+      </div>
+      <div className="hamburger">
+        <div className="line1" />
+        <div className="line2" />
+        <div className="line3" />
+      </div>
+      <ul className="nav-links">
+        {links.map((link) => (
+          <li key={link.id}>
+            <Link to={link.path}>{link.name}</Link>
+          </li>
```

Comment:
- [ ] Great job so far 👏🏻 But According to the project requirements 👇🏻 Let's make sure to use `<NavLink />` for the page navigation links instead of `<link />`. Kindly address this 💯 👍🏻 

![18](https://user-images.githubusercontent.com/79058364/193143667-aa1a6320-92f2-4ee2-8bcf-e7c0303e3c54.png)

Reply:
Fixed